### PR TITLE
[BBL-34] Create token and stake contract

### PR DESCRIPTION
**Pull request by 0xmeetdev**

### CODE REVIEW

File: src/Stack.sol
Comment (0xblackdevil) [inline src/Stack.sol:1]: add suspendStake function in stake.sol file

**Comment by 0xmeetdev** [comment]
### Temp Comment for fatching it

Change it one more time

**Comment by 0xmeetdev** [comment]
I think I have to generate new api call

**Comment by 0xmeetdev** [comment]
another comment for testing

**Pull request closed without merging**